### PR TITLE
Support environment variables for server hostname, port options

### PR DESCRIPTION
**Pull request by GochoMugo**
Feature:

- Supports the following environment variables:
  + **REDIS_HOST**: Server hostname
  + **REDIS_PORT**: Server port
- CLI options override the above environment variables

### CODE REVIEW

File: redis-cli.go [hunk 1]
```diff
@@ -41,6 +42,12 @@ const (
 func main() {
 	flag.Parse()
 
+	portInt, err := strconv.Atoi(*portStr)
+	if err != nil {
+		fmt.Printf("(error) %s", err.Error())
+	}
+	port = &portInt
```
Comment (holys) [inline redis-cli.go]: how about using `portStr` directly? 
https://github.com/holys/redis-cli/blob/88d832719fa6e915be4438bbdc686936338396d1/redis-cli.go#L275

Comment (GochoMugo) [reply redis-cli.go]: Addressed in commit https://github.com/holys/redis-cli/pull/9/commits/0b0d2b2edf84ea5121a30b19a2147ae01de51856.

**Comment by holys** [comment]
LGTM. Thank you!

**Comment by GochoMugo** [comment]
Thank you for this tool. Really helpful when I just need the client but not the server. :+1:

**Comment by holys** [comment]
Thank you for your contribution @GochoMugo  🌹

**Pull request merged**